packages.json

```diff
@@ -6162,5 +6162,22 @@
     "description": "A simple and lightweight terminal coloring library.",
     "license": "MIT",
     "web": "https://github.com/molnarmark/colorize"
+  },
+  {
+    "name": "cello",
+    "url": "https://github.com/unicredit/cello",
+    "method": "git",
+    "tags": [
+      "string",
+      "succint data structure",
```

Comment:
I have used dashes, hope it is good. Many words do not really stand by themselves. For instance, this library provides wavelet trees, but it has nothing to do with wavelets (it is a historical accident that the former are named after the latter)